### DISCUSSION TITLE
change colspan of td in details row

### DISCUSSION THREAD

**Discussion by blessenabrahamnw18** [komarovalexander/ka-table#281] [General]
<tr class="ka-tr ka-details-row"><td class="ka-cell" colspan="19">...</td></tr>

I want to change the colspan of td.ka-cell to colspan="9"; so that the details row doesn't stretch out for 19 columns.

**Comment by komarovalexander** [comment]
Hi @blessenabrahamnw18 I've just added 

```
childComponents: {
  detailsCell: {
    elementAttributes: () => ({
      colSpan: 1
    }),
  }
}
```

use version 7.10.0

**Reply by blessenabrahamnw18** [reply]
Thankyou, this worked while using 7.10.0

**Reply by blessenabrahamnw18** [reply]
Hi Alexander,
Can you add elementAttributes for the no Data row cell as well.
Need to change td.ka-no-data-cell colspan.
Thanks

**Reply by komarovalexander** [reply]
Hi @blessenabrahamnw18 done, use v7.11.0
https://github.com/komarovalexander/ka-table/releases/tag/7.11.0

**Reply by blessenabrahamnw18** [reply]
Thankyou @komarovalexander!!
